patch-package
--- a/node_modules/react-dnd-html5-backend/src/HTML5Backend.js
+++ b/node_modules/react-dnd-html5-backend/src/HTML5Backend.js
@@ -297,11 +297,15 @@ export default class HTML5Backend {
 		// Receiving a mouse event in the middle of a dragging operation
 		// means it has ended and the drag source node disappeared from DOM,
 		// so the browser didn't dispatch the dragend event.
-		this.window.addEventListener(
+		//
+		// We need to wait an execution frame before we start listening for mousemove events.
+		// This is needed because the drag preview needs to be drawn or else it fires an 'mousemove' event
+		// immediately in some browsers.
+		setTimeout((_) => _.window.addEventListener(
 			'mousemove',
 			this.endDragIfSourceWasRemovedFromDOM,
-			true,
-		)
+			true
+		), 0, [this])
 	}
 
 	clearCurrentDragSourceNode() {
